vendors/espressif/boards/esp32/aws_demos/config_files/FreeRTOSIPConfig.h

```diff
@@ -230,9 +235,10 @@ extern uint32_t ulRand();
 
 /* The MTU is the maximum number of bytes the payload of a network frame can
  * contain.  For normal Ethernet V2 frames the maximum MTU is 1500.  Setting a
- * lower value can save RAM, depending on the buffer management scheme used.  If
- * ipconfigCAN_FRAGMENT_OUTGOING_PACKETS is 1 then (ipconfigNETWORK_MTU - 28) must
- * be divisible by 8. */
+ * lower value can save RAM.
+ * When the distance between Wi-Fi station and access point is long, or when
+ * the signal passes through walls, packets may get lost.  In that case it
+ * helps to decrease the MTU. */
```

Comment:
I'm nervous about adding this recommendation here unless we accompany it with an explanation of why this is the case.

Reply:
+1, please remove the comments on lines 239-241.